### ISSUE TITLE
Add Per-World Spawning

### ISSUE THREAD

**Issue by QuinnCHall** [GC-spigot/AdvancedSpawners#350]
Right now I can add a list of worlds that all custom mobs can spawn in. However, it would be nice to be able to specify specific worlds for specific mobs. For example, if I were to make a fiery demon mob, I would want it to spawn specifically in the nether. But if I had a bird creature or some other creature that I wanted to spawn in the overworld, I couldn't have the two spawn in their specific worlds. The demon thing would spawn in the overworld, and the bird would spawn in the nether, because there is no way to specify where they should spawn.
Having the ability to set specific worlds for mobs created with AdvancedSpawners would allow users to greatly increase their creativity when it comes to new creatures in each dimension. If you feel like taking this a step further, instead of specifying just the worlds, you could possibly even add support for individual biomes inside of that world. However, I don't think that is as needed.

To summarize, I would love the ability to specify which world (using world names) specific creatures that I make in AdvancedSpawners to randomly spawn in.
